src/dsp_tools/commands/xmlupload/xmlupload.py

```diff
@@ -310,6 +310,7 @@ def _handle_permanent_connection_error(err: PermanentConnectionError) -> Never:
     msg = "Lost connection to DSP server, probably because the server is down. "
     msg += f"Please continue later with 'resume-xmlupload'. Reason for this failure: {err.message}"
     logger.error(msg)
+    msg += f"\nSee {WARNINGS_SAVEPATH} for more information."
```

Comment:
Previously, this was further down in the call hierarchy (in the connection_live.py), and was only visible in the log file and warnings.log file, which is of course useless.

I identified this place as being more appropriate: Here, it is printed to the Terminal, which is useful.